### ISSUE TITLE
Docker help ?

### ISSUE THREAD

**Issue by tablatronix** [ct-Open-Source/tuya-convert#829]
Ok, I am not very skilled in docker but this throwing me for a loop

Followed the instructions, 
What is up with the container name and service?

![Screen Shot 2020-11-25 at 7 15 01 AM](https://user-images.githubusercontent.com/807787/100232503-0a9baa00-2eee-11eb-8b7a-87808031035f.png)

using debian, image

it runs ok with run, but then logging and interacting with the container to debug the AP issue , and I have no logging

**Comment by tablatronix** [comment]
I tried adding a static container name to the config yml, same thing..

**Comment by tablatronix** [comment]
ohh Do i need to run this in the docker folder after build?

**Comment by tablatronix** [comment]
It works now, I built against 
`FROM phusion/baseimage:latest-arm`

Now I have to fix the dnsmasq issue

**Comment by tablatronix** [comment]
Finally got it to work!
never did get the screen logging working shrug..

it looks like the script needs to have some better start ap checking, I found out that the dns and ap were actually working , but crashing, sometimes it would show up for a bit but crash when I connected to it. I eventually got it to stay up by killing dnsmasq and hostapd that seemed to be stuck running. Ill look into it further to see what actually fixed it. Running start ap outside of docker seemed to help me get some better tests

![Screen Shot 2020-11-26 at 12 00 28 AM](https://user-images.githubusercontent.com/807787/100364169-5ec48e00-2fc3-11eb-81e4-b9a620f6a003.png)
